code/game/objects/items/weapons/material/knives.dm

```diff
@@ -1,23 +1,90 @@
-/obj/item/weapon/material/butterfly
+/*
+ * Knives. They stab your eyes out, and fit into boots. Copypasted the screwdriver code
+ */
+/obj/item/weapon/material/knife
+	name = "kitchen knife"
+	icon = 'icons/obj/kitchen.dmi'
+	icon_state = "knife"
+	desc = "A general purpose Chef's Knife made by SpaceCook Incorporated. Guaranteed to stay sharp for years to come."
+	flags = CONDUCT
+	sharp = 1
+	edge = 1
+	var/active = 1 // For butterfly knives
+	force_divisor = 0.15 // 9 when wielded with hardness 60 (steel)
+	matter = list(DEFAULT_WALL_MATERIAL = 12000)
+	origin_tech = list(TECH_MATERIAL = 1)
+	attack_verb = list("slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
+	unbreakable = 1
+
+/obj/item/weapon/material/knife/attack(mob/living/carbon/M as mob, mob/living/carbon/user as mob, var/target_zone)
+	if(active == 1)
+		if(target_zone != "eyes" && target_zone != "head")
+			return ..()
+		if((CLUMSY in user.mutations) && prob(50))

```

Comment:
While entertaining, the CLUMSY tag hasn't been used since clowns were removed, to my knowledge.  I'm also not sure how someone is capable of stabbing themselves in the eye when trying to stab someone else, even with a 50% chance.  This bit of code is kind of unnecessary and a silly meme, might be better removed.